include/aws/s3/private/s3_request.h

```diff
@@ -17,7 +17,8 @@ struct aws_s3_meta_request;
 
 enum aws_s3_request_desc_flags {
     AWS_S3_REQUEST_DESC_RECORD_RESPONSE_HEADERS = 0x00000001,
-    AWS_S3_REQUEST_DESC_PART_SIZE_RESPONSE_BODY = 0x0000002,
+    AWS_S3_REQUEST_DESC_PART_SIZE_RESPONSE_BODY = 0x00000002,
+    AWS_S3_REQUEST_DESC_ALWAYS_SEND = 0x00000004,
```

Comment:
Irrelevant: What does desc stand for here?

Reply:
It's left over from when there used to a be an aws_s3_request_desc structure, ie, "AWS S3 Request Description".  I have removed it!